src/_pytest/_code/code.py

```diff
@@ -41,7 +41,7 @@
 
     from _pytest._code import Source
 
-    _TracebackStyle = Literal["long", "short", "no", "native"]
+    _TracebackStyle = Literal["long", "short", "line", "no", "native"]
```

Comment:
"native" in not used internally with `FormattedExcinfo` at least (https://github.com/blueyed/pytest/blob/f1609d6e956bdf4d712610d8e6ca9fbb0b2ed108/src/_pytest/_code/code.py#L612-L631), so it might make sense to have two different types here (but it makes sense to match the `--tb` option here I guess (currently (with this patch) missing "auto")) - but that would be a followup.

Reply:
Should `line` be added to `ExceptionInfo.getrepr()` docstring?

Reply:
I do not think it is used there - i.e. I am not sure.
I think it is only used here: https://github.com/pytest-dev/pytest/blob/719e0e354bd9e13c5a9ecc3f1ac9839d3e7a8704/src/_pytest/terminal.py#L868